.github/actions/maintainer-check/action.yaml

```diff
@@ -9,7 +9,7 @@ runs:
   steps:
     - shell: bash
       run: |
-        MAINTAINERS="tzneal,spring1843,ellistarn,akestner,bwagner5,billrayburn,suket22,njtran,chrisnegus,jonathan-innis,rtripat,engedaam"
+        MAINTAINERS="tzneal,ellistarn,akestner,bwagner5,billrayburn,suket22,njtran,chrisnegus,jonathan-innis,rtripat,engedaam"
```

Comment:
musing: We really shouldn't be doing the check this way. This has so much potential to get way out of date.